src/Components/test/testassets/BasicTestApp/EventCustomArgsComponent.razor

```diff
@@ -0,0 +1,102 @@
+﻿@using BasicTestApp.CustomEventTypesNamespace
+
+<h3 id="event-custom-args">Custom event types</h3>
+
+<p>This component exercises various scenarios around custom event types and arguments.</p>
+
+<div id="test-event-target" style="border: 1px solid black; background: silver; padding: 1em; margin-bottom: 1em;"
+     @onkeydown="@(e => { LogMessage("Received native keydown event"); })"
+     @ontestevent="@HandleTestEvent"
+     @onkeydown.testvariant="@HandleCustomKeyDown"
+     @onkeydown.yetanother="@HandleYetAnotherKeyboardEvent"
```

Comment:
What's the invocation order for callbacks when you have multiple registered event handlers like this for the same event?

Reply:
Invocation order is unaffected by this PR.

I don't think we've ever formally promised that it will run in a specific order.

Reply:
It's basically like native DOM events, in that they occur in some particular order in practice, but you shouldn't rely on it: https://stackoverflow.com/questions/282245/dom-event-precedence. For native DOM events, it may vary across browsers.

In our implementation it would fire the native event first, and then the aliases would fire in order based on registration. But since browsers themselves don't define this officially, I hope we don't need to either.

Reply:
Sounds good!

I was asking since we use lexical order for things like `@splat` attributes and wanted to make sure that if there was such thing in the DOM we considered respecting it.